### DISCUSSION TITLE
Weird tag behavior between MacOS and iOS

### DISCUSSION THREAD

**Discussion by apastuszak** [glushchenko/fsnotes#1561] [General]
I have a bunch of notes tagged with `#keto` and `#recipe`. On MacOS, when I click on this tag, I see all 11 keto recipes that I have.

On iOS, when I click on the `#keto` or `#recipe` tag, I only see 3 of my recipes.

The other recipes are on iOS. They just don't show up when I click on the tag on the left-hand side.

**Comment by gingerbeardman** [comment]
Are you selecting the same folder across both?

Search is limited to the currently selected folder.

**Comment by apastuszak** [comment]
Today it works. I just launched the app on my iPhone and I'm seeing all my keto notes.  Very odd.
